smartapps/smartthings/ecobee-connect.src/ecobee-connect.groovy

```diff
@@ -381,75 +384,41 @@ def pollHandler() {
 }
 
 def pollChildren(child = null) {
-	def thermostatIdsString = getChildDeviceIdsString()
-	log.debug "polling children: $thermostatIdsString"
-	def data = ""
+    def thermostatIdsString = getChildDeviceIdsString()
+    log.debug "polling children: $thermostatIdsString"
+
+    def requestBody = [
```

Comment:
I'm very supportive of this, because hardcoding the JSON makes the code hard to read, and even harder when you need to add more params.  Have you tested to make sure it works?  It definitely should, but I would just want to verify it